app/views/registrations/_form.html.haml

```diff
@@ -2,6 +2,7 @@
 = simple_form_for([@event, @registration], validate: true) do |f|
   = f.input :started_at, as: :hidden, input_html: { value: @registration.started_at }
   = f.input :step_to_validate, as: :hidden, input_html: { value: "start" }
+  = f.input, :user_id, as: :hidden, input_html: { value: current_user.try(:id) }
```

Comment:
Looks good! I was getting a syntax error on this line for the comma after `= f.input`. Did that make its way in on accident?